pkg/util/containers/types.go

```diff
@@ -47,6 +47,13 @@ const (
 	UnknownNetworkMode        = "unknown"
 )
 
+// Container UTS modes
+const (
+	DefaultUTSMode string = ""
+	HostUTSMode           = "host"
+	UnknownUTSMode        = "unknown"
+)
```

Comment:
Defining a type for UTS mode can be interesting.
```
type UTSMode string
const (
	DefaultUTSMode  UTSMode = ""
	HostUTSMode             = "host"
	UnknownUTSMode          = "unknown"
)
```

Reply:
Done!